.eslintignore

```diff
@@ -3,3 +3,4 @@ build
 reports
 lib
 **/aggregated-translations/*.*
+dist
```

Comment:
added dist to ignore globally, webpack defaults to the dist dir instead of build as we have it configured.

Reply:
This is included because as part of the tests, webpack is run and creates a dist folder.